File: leetcode/two-sum.go
func twoSum(nums []int, target int) []int {
    for x := 0 ; x < len(nums) ; x++ {
        for y := x+1 ; y < len(nums) ; y++ {
            if nums[x]+nums[y]==target && y!=x {
                return []int{x, y}
            }
        }
    }
    return []int{};
}

File: leetcode/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> fo;
        for(int i = 0;i<nums.size();++i){
            for(int x=i+1;x<nums.size();++x){
                if(nums[x]+nums[i]==target){
                    fo.push_back(i);
                    fo.push_back(x);
                }
            }
        }
        return fo;
    }
};